README.md

```diff
@@ -172,6 +172,47 @@ The previous configuration will expect and build the files:
 - `src/index.html` => `dist/index.html`
 - `src/index.js` => `dist/index.js`
 
+#### Excluding a page from chunks
+
+If you want a page to be excluded from either the [vendor](#vendor) or [common](#common) chunk, then you can do so by providing an object with a `name` and `independent` flag (set to `true`) instead of just the name of the page.
+
+```js
+// sagui.config.js
+module.exports = {
+  pages: ['index', { name: 'about', independent: true }]
```

Comment:
Very small suggestion, but what about using a `demo` or `playground` in the name, to better hint this could be used for development pages.

Reply:
Makes sense. Will fix it.